.github/workflows/release.yml

```diff
@@ -34,58 +31,10 @@ jobs:
         if: contains(fromJson('["release", "build", "close"]'), github.workflow)
       - name: Install dependencies
         run: yarn install --check-files --frozen-lockfile
-      - name: release
-        run: yarn turbo run release --filter=functionless
-      - name: Check for new commits
-        id: git_remote
-        run: echo ::set-output name=latest_commit::"$(git ls-remote origin -h ${{ github.ref }} | cut -f1)"
-      - name: Upload artifact
-        if: ${{ steps.git_remote.outputs.latest_commit == github.sha }}
-        uses: actions/upload-artifact@v2.1.1
-        with:
-          name: build-artifact
-          path: dist
-  release_github:
-    name: Publish to GitHub Releases
-    needs: release
-    runs-on: ubuntu-latest
-    permissions:
-      contents: write
-    if: needs.release.outputs.latest_commit == github.sha
-    steps:
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 14.x
-      - name: Download build artifacts
-        uses: actions/download-artifact@v3
-        with:
-          name: build-artifact
-          path: dist
-      - name: Release
-        run: errout=$(mktemp); gh release create $(cat dist/releasetag.txt) -R $GITHUB_REPOSITORY -F dist/changelog.md -t $(cat dist/releasetag.txt) --target $GITHUB_REF 2> $errout && true; exitcode=$?; if [ $exitcode -ne 0 ] && ! grep -q "Release.tag_name already exists" $errout; then cat $errout; exit $exitcode; fi
+      - run: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > .npmrc
```

Comment:
Just checking: does this result in the token being in the logs? I don't think so but worth a check

Reply:
It shouldn't and doesn't appear to be from the test runs.